apps/web/vibes/soul/primitives/animated-link/index.tsx

```diff
@@ -12,8 +11,15 @@ export interface AnimatedLinkProps {
 
 export const AnimatedLink = function AnimatedLink({ href, label, className }: AnimatedLinkProps) {
   return (
-    <Link href={href} className={clsx('animated-link animated-underline', className)}>
-      <span>{label}</span>
+    <Link
+      href={href}
+      className={clsx(
+        'origin-left pb-0.5 font-semibold transition-[background-size] duration-500 [background:linear-gradient(0deg,hsl(var(--primary)),hsl(var(--primary)))_no-repeat_left_bottom_/_0_var(--bg-h)] hover:bg-[size:100%_var(--bg-h)] focus:outline-none focus-visible:bg-[size:100%_var(--bg-h)]',
+        className
+      )}
+      style={{ '--bg-h': '2px' } as CSSProperties}
```

Comment:
Can't we use a computed property in the Tailwind classes? How come we pass it via an inline style if it doesn't depend on state?

Reply:
Just went ahead and removed the css variable since it's only used in this component in very few places.